3ds Max/Max2Babylon/Exporter/BabylonExporter.Mesh.cs

```diff
@@ -121,11 +121,15 @@ private BabylonNode ExportMesh(IIGameScene scene, IIGameNode meshNode, BabylonSc
                         // Check if we need to export this instance as an instance mesh.
                         if (tabGuid == masterMeshPair.Key.id)
                         {
-                            // If there is already an exported mesh in the scene that shares this mesh's material, then export it as an instance.
-                            if (masterMeshPair.Key.materialId == null
-                            || (meshNode.NodeMaterial != null && meshNode.NodeMaterial.MaxMaterial.GetGuid().ToString().Equals(masterMeshPair.Value.NodeMaterial.MaxMaterial.GetGuid().ToString())))
+                            bool isShareMat = masterMeshPair.Key.materialId == null || (meshNode.NodeMaterial != null && meshNode.NodeMaterial.MaxMaterial.GetGuid().ToString().Equals(masterMeshPair.Value.NodeMaterial.MaxMaterial.GetGuid().ToString()));
+
+                            BabylonNode n = isShareMat?
+                                ExportInstanceMesh(scene, meshNode, babylonScene, masterMeshPair.Key, masterMeshPair.Value) :
```

Comment:
The use of a ternary operator makes this code hard to understand at first glance. Can we just use if-else for readability here?